metric-anomaly-task-manager/src/main/java/org/hypertrace/alert/engine/metric/anomaly/task/manager/rulesource/RuleSourceProvider.java

```diff
@@ -0,0 +1,23 @@
+package org.hypertrace.alert.engine.metric.anomaly.task.manager.rulesource;
```

Comment:
nit: package could be `...rule.source` so that other rule related classes can go inside `...rule` package

Reply:
Done.